index.html

```diff
@@ -10507,13 +10510,17 @@ <h3>User Prompt Handler</h3>
    set <var>handler<var> to "<code>dismiss</code>"
    and <var>notify</var> to true.
 
+   <li><p>If <var>handler</var> is "<code>ignore</code>",
```

Comment:
So if the handler is already "ignore", we set it to "ignore" again?

Reply:
Ah good point. I should not have simply copy and pasted. Yes, we only need to set `notify` to `true` here.